src/bin/cql-stress-cassandra-stress/settings/command/help.rs

```diff
@@ -13,24 +13,24 @@ pub fn print_help() {
     Options::print_generic_help();
 }
 
-fn print_command_help(command_str: &str) -> Result<CommandParams> {
+fn print_command_help(command_str: &str) -> Result<Option<CommandParams>> {
     match Command::parse(command_str) {
         Ok(cmd) => {
             cmd.print_help();
-            Ok(CommandParams::Special)
+            Ok(None)
         }
         Err(_) => Err(anyhow::anyhow!(
             "Invalid command or option provided to command help"
         )),
     }
 }
 
-fn print_option_help(option_str: &str) -> Result<CommandParams> {
+fn print_option_help(option_str: &str) -> Result<Option<CommandParams>> {
     Options::print_help(option_str)?;
-    Ok(CommandParams::Special)
+    Ok(None)
 }
```

Comment:
I'm not sure if I understand why this function returned `Result<CommandParams>` in the first place. It is supposed to only print help, right? Returning `Result<>` I can understand, but `CommandParams`?